src/CommunityToolkit.Maui/Converters/ICommunityToolkitValueConverter.shared.cs

```diff
@@ -52,7 +52,7 @@ public interface ICommunityToolkitValueConverter : IValueConverter
 
 
 	/// <inheritdoc />
-#if NET8_0 // Should be fixed in .NET 9
+#if NET8_0 || NET9_0 // Should be fixed in .NET 9 - Not fixed in the first RC
```

Comment:
Dang - we should open an Issue / PR ASAP on the .NET MAUI repo for this and work with @jfversluis to ensure it gets fixed before .NET 9 GAs. 

This is the problem I highlighted to Gerald in our last standup when I was going over the [AOT PR](https://github.com/CommunityToolkit/Maui/pull/1820).

Reply:
Long story short, the .NET MAUI team just needs to add the following `[DynamicallyAccessedMembers]` attribute to ensure `IValueConverter` is AOT compatible:

```cs
public interface IValueConverter
{
    object? Convert(object? value, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] Type targetType, object? parameter, CultureInfo culture);

    object? ConvertBack(object? value, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] Type targetType, object? parameter, CultureInfo culture);
}
```

Reply:
I would also recommend that the .NET MAUI team enable `<warningsaserrors>` for all of the Linker + AOT warnings to ensure they don't miss any before GA, similar to how we have in [`Directory.Build.props`](https://github.com/CommunityToolkit/Maui/blob/eeff00e99bb815195d84262c00f2f229e1679647/Directory.Build.props#L20-L69)

Reply:
Good news! I confirmed that everything in the .NET MAUI repo is good, from a trimming standpoint. 

We're getting this is error is that the MAUI team is not yet using nullable reference types everywhere yet (crazy - right??), yet we are. So we are just getting yelled at for using `object?` instead of `object`.